src/riot-rs/src/identity.rs

```diff
@@ -0,0 +1,32 @@
+//! Access to unique identifiers provided by the device.
```

Comment:
This is the first actual code in the  `riot-rs` crate. Also, the first that adds docs.
`random`and `storage` get their own crate. Maybe move this into e.g., `riot-rs-identity` as well?